libsolutil/StringUtils.h

```diff
@@ -101,109 +101,113 @@ std::string joinHumanReadablePrefixed
 		return _separator + joinHumanReadable(_list, _separator, _lastSeparator);
 }
 
-/// Same as @ref formatNumberReadable but only for unsigned numbers
+/// Formats large numbers to be easily readable by humans.
+/// Returns decimal representation for smaller numbers; hex for large numbers.
+/// "Special" numbers, powers-of-two and powers-of-two minus 1, are returned in
+/// formulaic form like 0x01 * 2**24 - 1.
+/// @a T can be any integer variable, will typically be u160, u256 or bigint.
```

Comment:
```suggestion
/// @a T can be any integer type, will typically be u160, u256 or bigint.
```
Ignore upper case letter `A`, it should be `a`, but github is somehow tagging a user there... 
Just to be clear, suggestion is to change `variable` to `type`